src/app/component-demos/page-layouts/sidebar-page-layout/sidebar-page-layout.component.ts

```diff
@@ -5,7 +5,8 @@ import { Component } from '@angular/core';
   templateUrl: './sidebar-page-layout.component.html',
 })
 export class SidebarPageLayoutComponent {
-  sidebarPageLayout = `<mad-main-container>
+  sidebarPageLayout = `
```

Comment:
Wouldn't it be better to have this stuff inside a .html file? (To increase readability?)

Reply:
Yes, but I'll do it when I'll have a solution for syntax highlight for code snippets. Not a part of this MR